react/components/EXPERIMENTAL_Table/Sections/LoadedView.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC, PropsWithChildren } from 'react'
+
+import { useLoadingContext } from '../context/loading'
+
+function LoadedView({ children }: PropsWithChildren<{}>) {
+  const { empty, loading } = useLoadingContext()
+  return !empty && !loading ? <>{children}</> : null
```

Comment:
```suggestion
  return !empty && !loading ? <Fragment>{children}</Fragment> : null
```

Reply:
Why not just `children`? 🤔

Reply:
I tried with just `children` but the typescript don't recognize the function as a component, when I try using `yarn styleguide` it works, but don't pass by the type checking on `vtex link`. I also tried to type the function as `FC` directly on the declaration, but the typescript also says the return type is not a component